server/types/User.ts

```diff
@@ -8,3 +8,14 @@ export type UserProfile = Pick<User, 'id'
   | 'linkedin_url'
   | 'github_username'
   | 'website'>;
+
+export interface UserObject {
+  email: string;
+  discord_user_id: string;
+  discord_name: string;
+  bio: string;
+  twitter_username: string;
+  linkedin_url: string;
+  github_username: string;
+  website: string;
+}
```

Comment:
i think this is basically the `User` type without the `id` field; you can use `Omit` to perform the opposite of `Pick` to remove fields:

```typescript
export type UserFields = Omit<User, 'id'>;
```

Reply:
so would it be better for me to use the User type and omit? or leave as is?

Reply:
I'd use omit, less maintenance and less redundancy

Reply:
i think I'm having a typescript issue, the new Omit export isnt working for these two lines of code

function createManyUsers(numberOfUsers: number): UserObject[] {
  return Array.from({ length: numberOfUsers }, makeUnsavedUserModel);
}

Reply:
nvm i got it!